### PR TITLE
SPUI-1218 > Fix copied in transaction history

### PR DESCRIPTION
**Pull request by DenisMartych**

### CODE REVIEW

File: lib/app/shared/features/wallet/helper/transaction_item.dart [hunk 1]
```diff
@@ -137,4 +142,11 @@ class TransactionItem extends HookWidget {
...
     );
   }
+
+  bool _displayTransformTranslate() {
+    return transactionListItem.operationType !=
+            OperationType.paidInterestRate &&
+        transactionListItem.operationType != OperationType.rewardPayment &&
+        transactionListItem.operationType != OperationType.receiveByPhone;
+  }
 }
```
Comment (eli1stark) [inline lib/app/shared/features/wallet/helper/transaction_item.dart:152]: move `transaction_item.dart` under `view/components folder`

File: lib/app/shared/features/wallet/view/components/wallet_body/components/transactions_list_item/components/transaction_details/components/common_transaction_details_block.dart [hunk 2]
```diff
@@ -46,7 +46,10 @@ class CommonTransactionDetailsBlock extends HookWidget {
             color: colors.grey2,
           ),
         ),
-        if (transactionListItem.operationType != OperationType.paidInterestRate)
+        if (transactionListItem.operationType !=
```
Comment (eli1stark) [inline lib/app/shared/features/wallet/view/components/wallet_body/components/transactions_list_item/components/transaction_details/components/common_transaction_details_block.dart:49]: use isOperationHistoryItemSupportCopy

**Review by eli1stark — CHANGES_REQUESTED** [review]

File: lib/app/shared/features/wallet/helper/transaction_item.dart [hunk 3]
```diff
@@ -137,4 +142,11 @@ class TransactionItem extends HookWidget {
       ],
     );
   }
+
+  bool _displayTransformTranslate() {
```
[the remark this answers is not in the document]
Comment (DenisMartych) [reply lib/app/shared/features/wallet/helper/transaction_item.dart:146]: done

[the remark this answers is not in the document]
Comment (DenisMartych) [reply lib/app/shared/features/wallet/helper/transaction_item.dart:146]: done

File: lib/app/shared/features/wallet/helper/transaction_item.dart (hunk 1 above)
Comment (DenisMartych) [reply lib/app/shared/features/wallet/helper/transaction_item.dart:152]: done

File: lib/app/shared/features/wallet/view/components/wallet_body/components/transactions_list_item/components/transaction_details/components/common_transaction_details_block.dart (hunk 2 above)
Comment (DenisMartych) [reply lib/app/shared/features/wallet/view/components/wallet_body/components/transactions_list_item/components/transaction_details/components/common_transaction_details_block.dart:49]: done

**Pull request merged**